src/deprecated/genentropy.jl

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
In this file you just have to do `@deprecate genentropy entropy_renyi` :D

Reply:
Ah, that makes everything a lot easier 👍